app/views/flood_risk_engine/enrollments/steps/show.html.erb

```diff
@@ -3,10 +3,21 @@
 <div class="grid-row">
   <div class="column-two-thirds">
 
-    <%= render(
-          "flood_risk_engine/enrollments/header",
-          title_text: t("flood_risk_engine.enrollments.steps.#{step}.heading")
-        )
+    <% unless form.no_header_in_show %>
+      <%= render(
+              "flood_risk_engine/enrollments/header",
+              title_text: t("flood_risk_engine.enrollments.steps.#{step}.heading")
+          )
+      %>
+    <% end %>
+
+    <%= render 'flood_risk_engine/enrollments/validation_errors', form: form %>
+
+    <% # Using 'as:' changes the name of the fields in the form from eg
```

Comment:
This puts the comment code about `as:` back in after it was replaced with a shorter comment on the line using the `:as` (line 25). If you prefer the commend at it was, can you please remove the comment on line 25.